apps/mobile/src/app/(home)/account/[account]/index.tsx

```diff
@@ -12,11 +15,17 @@ const configureAccountParamsSchema = z.object({
 
 export default function AccountScreen() {
   const params = useLocalSearchParams();
+
   const { fingerprint, account: accountIndex } = configureAccountParamsSchema.parse(params);
+  const account = useAccountByIndex(fingerprint, accountIndex);
```

Comment:
I need to get `account` here to pass to `useGetTokensList`. 

Should I still use `<AccountLoader` below when passing to `AccountLayout`